packages/apex-node/src/streaming/streamingClient.ts

```diff
@@ -40,7 +40,9 @@ export class Deferred<T> {
 }
 
 export class StreamingClient {
-  private client: ApexFayeClient;
+  // This should be a Client from Faye, but I'm not sure how to get around the type
+  // that is exported from jsforce.
+  private client: any;
```

Comment:
So this is sadness but I'm not sure about the right path forward.  Basically the type for the Client class from jsforce doesn't include methods that are used in this class.  I tried lots of different ways to get the local typings file to alter the module exported by jsforce, but nothing worked.  My googling led me to believe that's a pretty old way to define typings (@types seems to be what's doc'd now).  I'm open to better suggestions here.